contracts/staking/contracts/src/stake/MixinStake.sol

```diff
@@ -158,36 +177,47 @@ contract MixinStake is
 
     /// @dev Delegates a owners stake to a staking pool.
     /// @param poolId Id of pool to delegate to.
-    /// @param owner who wants to delegate.
-    /// @param amount of stake to delegate.
+    /// @param owner Owner who wants to delegate.
+    /// @param amount Amount of stake to delegate.
     function _delegateStake(
         bytes32 poolId,
         address payable owner,
         uint256 amount
     )
         private
     {
-        // sanity check the pool we're delegating to exists
+        // Sanity check the pool we're delegating to exists.
         _assertStakingPoolExists(poolId);
 
-        // cache amount delegated to pool by owner
-        IStructs.StoredBalance memory initDelegatedStakeToPoolByOwner = _loadUnsyncedBalance(_delegatedStakeToPoolByOwner[owner][poolId]);
+        // Cache amount delegated to pool by owner.
+        IStructs.StoredBalance memory initDelegatedStakeToPoolByOwner =
+            _loadUnsyncedBalance(_delegatedStakeToPoolByOwner[owner][poolId]);
 
-        // increment how much stake the owner has delegated to the input pool
-        _incrementNextBalance(_delegatedStakeToPoolByOwner[owner][poolId], amount);
+        // Increment how much stake the owner has delegated to the input pool.
+        _incrementNextBalance(
+            _delegatedStakeToPoolByOwner[owner][poolId],
+            amount
+        );
 
-        // increment how much stake has been delegated to pool
+        // Increment how much stake has been delegated to pool.
         _incrementNextBalance(_delegatedStakeByPoolId[poolId], amount);
 
-        // synchronizes reward state in the pool that the staker is delegating to
-        IStructs.StoredBalance memory finalDelegatedStakeToPoolByOwner = _loadAndSyncBalance(_delegatedStakeToPoolByOwner[owner][poolId]);
-        _syncRewardsForDelegator(poolId, owner, initDelegatedStakeToPoolByOwner, finalDelegatedStakeToPoolByOwner);
+        // Synchronizes reward state in the pool that the staker is delegating
+        // to.
+        IStructs.StoredBalance memory finalDelegatedStakeToPoolByOwner =
+            _loadAndSyncBalance(_delegatedStakeToPoolByOwner[owner][poolId]);
+        _syncRewardsForDelegator(
```

Comment:
Are you using `prettier` or something similar here? Would love to get my hands on it!

Reply:
I just manually formatted  everything I touched to 80 characters. :smile: